ExceptionsDisabled.cpp

```diff
@@ -29,9 +29,11 @@ namespace eqlib {
 
 // CListWnd
 #ifdef CListWnd__AddColumn1_x
-FUNCTION_AT_ADDRESS(int, CListWnd::AddColumn(const CXStr& Label, CTextureAnimation* pTA, int Width, uint32_t Flags,
-	CXStr Tooltip, uint32_t Type, CTextureAnimation* pTASelected, CTextureAnimation* pTAMouseOver, bool bResizeable,
-	CXSize TextureSize, CXPoint TextureOffset), CListWnd__AddColumn1);
+/* FUNCTION_AT_ADDRESS(int, CListWnd::AddColumn(const CXStr& Label, CTextureAnimation* pTA, int Width, uint32_t Flags,
+ *	CXStr Tooltip, uint32_t Type, CTextureAnimation* pTASelected, CTextureAnimation* pTAMouseOver, bool bResizeable,
+ *	CXSize TextureSize, CXPoint TextureOffset), CListWnd__AddColumn1);
+*/
+FUNCTION_AT_ADDRESS(int, CListWnd::AddColumn(const CXStr& Label, CTextureAnimation* pTA, int Width, uint32_t Flags, CXStr Tooltip, uint32_t Type, CTextureAnimation* pTASelected, CTextureAnimation* pTAMouseOver, bool bResizeable, CXSize TextureSize, CXPoint TextureOffset), CListWnd__AddColumn1);
```

Comment:
what change am i looking at here?

Reply:
Me being lazy about not scripting the check to see if a function spans multiple lines (but wanting to leave the pretty formatting for reference)